src/web/src/alert_rules_list.ecpp

```diff
@@ -196,71 +196,104 @@ if (streq (part, "LIST")) {
     std::string assetLink = "";
     std::string externalName = "";
 
+    log_debug ("Collecting the data:");
+
     while (part) {
+        
         const char *internalCat = "\"CAT_INTERNAL\"";
 
         //we need to serialize the section
         if (strstr(part, internalCat) == nullptr) {
 
-            //format to json minified, that's not optimal but that's the best way
-            // to ensure the good format to work with the regex
-            // minify => deserialize and serialize it again
-            std::stringstream inputTmp(part.get());
-            cxxtools::JsonDeserializer deserializerTmp(inputTmp);
-            
-            cxxtools::SerializationInfo siTmp;
-            deserializerTmp.deserialize(siTmp);
-
-            std::stringstream output;
-            cxxtools::JsonSerializer serializerTmp(output);
-            serializerTmp.beautify(false);
-            serializerTmp.serialize(siTmp).finish();
+            std::string data(part.get());
+            //log_debug ("Part: %s \n\n", data.c_str());
```

Comment:
could switch to log_trace() if it's really verbose and too much for debug, though still possibly useful!